### ISSUE TITLE
How to think about the fact that H5 released by Apple store is not allowed to be put on the market, does it have an impact on native script?

### ISSUE THREAD

**Issue by djroad** [react-navigation/react-navigation#6504]
How to think about the fact that H5 released by Apple store is not allowed to be put on the market, does it have an impact on native script?

**Comment by vonovak** [comment]
hello, I'm sorry, can you rephrase your question? I have no idea what you're talking about TBH :)
